uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/POMServiceImpl.java

```diff
@@ -58,6 +58,8 @@
 public class POMServiceImpl
         implements POMService {
 
+    public static final String POM_XML = "pom.xml";
```

Comment:
looks like unused variable POM_XML

Reply:
It is being used in WorkspaceProjectServiceImpl.java, lines 327 and 328.

Reply:
the literal "pom.xml" is using on same file another places also.Please use the constant there as well

Reply:
Changing code where I originally haven't touched could lead to unwanted conflicts, but I've applied the changes you've requested. Please check my last commit.